src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -174,27 +177,37 @@ class JesBackend extends Backend with LazyLogging {
 
     val outputMappings = call.task.outputs.map { taskOutput =>
       val rawValue = taskOutputToRawValue(taskOutput)
-      // FIXME: I stopped modifying here
       logger.debug(s"JesBackend setting ${taskOutput.name} to $rawValue")
       taskOutput.name -> rawValue
     }.toMap
 
-    status match {
-      case Run.Success(created, started, finished) =>
-        val taskOutputEvaluationFailures = outputMappings.filter {_._2.isFailure}
-        if (taskOutputEvaluationFailures.isEmpty) {
-          val unwrappedMap = outputMappings.collect { case (name, Success(wdlValue) ) => name -> wdlValue }
-          Success(unwrappedMap)
-        } else {
-          val message = taskOutputEvaluationFailures.collect { case (name, Failure(e))  => s"$name: $e" }.mkString("\n")
-          Failure(new Throwable(s"Workflow ${workflowDescriptor.id}: $message"))
-        }
-      case Run.Failed(created, started, finished, errorCode, errorMessage) =>
-        Failure(new Throwable(s"Workflow ${workflowDescriptor.id}: errorCode $errorCode for command: $commandLine. Message: $errorMessage"))
+    val warnAboutStderrLength: BigInteger =
+      if (!call.failOnStderr) BigInteger.valueOf(0)
+      else JesConnection.storage.objectSize(GoogleCloudStoragePath(stderrJesOutput(callGcsPath).gcs))
+
+    if (warnAboutStderrLength.intValue > 0) {
+      Failure(new Throwable(s"Workflow ${workflowDescriptor.id}: stderr has length $warnAboutStderrLength for command: $instantiatedCommandLine"))
+    } else {
+      status match {
+        //
+        case Run.Success(created, started, finished) =>
+          unwrapOutputValues(outputMappings, workflowDescriptor)
+        case Run.Failed(created, started, finished, errorCode, errorMessage) =>
+          Failure(new Throwable(s"Workflow ${workflowDescriptor.id}: errorCode $errorCode for command: $instantiatedCommandLine. Message: $errorMessage"))
+      }
     }
+  }
 
+  def unwrapOutputValues(outputMappings: Map[String, Try[WdlValue]], workflowDescriptor: WorkflowDescriptor): Try[Map[String, WdlValue]] = {
+    val taskOutputEvaluationFailures = outputMappings.filter { _._2.isFailure }
+    if (taskOutputEvaluationFailures.isEmpty) {
+      Success(outputMappings.collect { case (name, Success(wdlValue)) => name -> wdlValue })
+    } else {
+      val message = taskOutputEvaluationFailures.collect { case (name, Failure(e)) => s"$name: $e" }.mkString("\n")
```

Comment:
Note sure if this was discussed offline, but I believe `case` as a partial function always has to be enclosed in `{}`, not `()`. Working backwards from there, the choice on this line is only between what's there now, or the preferred removal of the infix `.`. Or I guess one could do `.({ case … })` but that might really spark more grumbling.

This does not apply to the non-`case` line 202 above, which could swap `. {}` for `.()`.